makefile

```diff
@@ -1,21 +1,25 @@
 NUM_NODES = 10
 WORLD_SIZE = 1000
 
-COMMON_CFLAGS = -g -std=gnu99 -O2 -mcpu=native -fomit-frame-pointer -Wall -Wextra
+COMMON_CFLAGS = -std=gnu99 -O2 -march=native -fomit-frame-pointer -Wall -Wextra
```

Comment:
-march implies -mtube, see https://gcc.gnu.org/onlinedocs/gcc-4.3.2/gcc/i386-and-x86_002d64-Options.html